autoload/ale/handlers/redpen.vim

```diff
@@ -1,5 +1,21 @@
 " Author: rhysd https://rhysd.github.io
 " Description: Redpen, a proofreading tool (http://redpen.cc)
+"
+function! ale#handlers#redpen#ResetOptions() abort
+    call ale#Set('redpen_options', '')
+endfunction
+
+" Reset the options so the tests can test how they are set.
+call ale#handlers#redpen#ResetOptions()
+
+function! ale#handlers#redpen#GetCommand(buffer) abort
+    let l:options = ale#Var(a:buffer, 'redpen_options')
+    let l:filetype = getbufvar(a:buffer, '&filetype')
```

Comment:
`&filetype` could be set to multiple types at once, for example `markdown.special` or some other construction. You can instead define the command with a lambda `{b -> string}` And you can apply `ale#Pad(ale#Var(b, 'redpen_options')` as `ale#Pad` will prefix `' '` if the value of the options variable is non-empty. In place of `l:filetype` here you can put in the value of `a:filetype` below from `ale#handlers#redpen#DefineLinter`.